build.gradle

```diff
@@ -2,7 +2,8 @@ group 'compiler-fleet'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.2.21'
+    ext.kotlin_version = '1.2.+'
+    ext.argparser_version = "2.+"
```

Comment:
This kinda excessive because you're unlikely to use it more than once

Reply:
Done